yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -60,6 +62,9 @@ private[yarn] class YarnAllocator(
 
   import YarnAllocator._
 
+  // RackResolver logs an INFO message whenever it resolves a rack, which is way too often.
+  Logger.getLogger(classOf[RackResolver]).setLevel(Level.WARN)
```

Comment:
Can you mention that in your first comment so its picked up as the commit message? (That's what I meant by "commit message", btw.)

Also, I'm pretty sure I've seen a bug to fix the config file issue. Might be worth it to look for it and mention it in the comments.